### PR TITLE
Added nodejs 4 and updated phantomjs and selenium.

### PR DESCRIPTION
**Pull request by maxnachlinger**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -26,12 +29,11 @@ RUN useradd -d /home/seleuser -m seleuser
...
 RUN chown -R seleuser /home/seleuser
 RUN chgrp -R seleuser /home/seleuser
-# fix https://code.google.com/p/chromium/issues/detail?id=318548
-RUN mkdir -p /usr/share/desktop-directories
+
 ADD ./scripts/ /home/root/scripts
-RUN npm install -g \
-  selenium-standalone@3.0.2 \
-  phantomjs@1.9.12 && \
+RUN npm i -g \
```
Comment (vvo) [inline Dockerfile]: can you keep the `install`, doesn't add anything to switch to `i`

File: Dockerfile [hunk 2]
```diff
@@ -7,7 +7,10 @@ MAINTAINER Vincent Voyer <vincent@zeroload.net>
 RUN apt-get -y update
 RUN apt-get install -y -q software-properties-common wget
 RUN add-apt-repository -y ppa:mozillateam/firefox-next
-RUN add-apt-repository -y ppa:chris-lea/node.js
+
+RUN wget -qO- https://deb.nodesource.com/setup_4.x | sudo bash -
```
Comment (vvo) [inline Dockerfile]: why not nodejs v5?

File: Dockerfile [hunk 3]
```diff
@@ -26,12 +29,11 @@ RUN useradd -d /home/seleuser -m seleuser
 RUN mkdir -p /home/seleuser/chrome
 RUN chown -R seleuser /home/seleuser
 RUN chgrp -R seleuser /home/seleuser
-# fix https://code.google.com/p/chromium/issues/detail?id=318548
-RUN mkdir -p /usr/share/desktop-directories
```
Comment (vvo) [inline Dockerfile]: Why remove this?

File: Dockerfile [hunk 4]
```diff
@@ -26,12 +29,11 @@ RUN useradd -d /home/seleuser -m seleuser
...
+
 ADD ./scripts/ /home/root/scripts
-RUN npm install -g \
-  selenium-standalone@3.0.2 \
```
Comment (vvo) [inline Dockerfile]: Why remove specific versions?

File: Dockerfile (hunk 1 above)
Comment (maxnachlinger) [inline Dockerfile]: That's true, just a habit of mine :)

File: Dockerfile (hunk 2 above)
Comment (maxnachlinger) [inline Dockerfile]: I figured the LTS version was a good choice for most users

File: Dockerfile (hunk 3 above)
Comment (maxnachlinger) [inline Dockerfile]: That bug has since been fixed.

File: Dockerfile (hunk 4 above)
Comment (maxnachlinger) [inline Dockerfile]: You're right, the dependencies were renamed but it's way more stable to use specific versions, I'll add those back.

File: Dockerfile (hunk 3 above)
Comment (vvo) [inline Dockerfile]: thanks

**Comment by vvo** [comment]
Thx for contribution, awesome work :)

**Pull request merged**